statistics/selectivity.go

```diff
@@ -79,11 +79,20 @@ func pseudoSelectivity(t *Table, exprs []expression.Expression) float64 {
 		}
 		switch fun.FuncName.L {
 		case ast.EQ, ast.NullEQ, ast.In:
+			minFactor = math.Min(minFactor, 1.0/pseudoEqualRate)
 			col, ok := t.Columns[colID]
-			if ok && (mysql.HasUniKeyFlag(col.Info.Flag) || mysql.HasPriKeyFlag(col.Info.Flag)) {
+			if !ok {
+				continue
+			}
+			if mysql.HasUniKeyFlag(col.Info.Flag) {
+				uniqueCol[col.Info.Name.L] = true
```

Comment:
do we need to set `uniqueCol` true when col has `PriKeyFlag`

Reply:
It is set at https://github.com/pingcap/tidb/pull/6302/files#diff-20ac3c94a9605bc56ae644dfd45569f8R94